js/__internal/grids/pivot_grid/field_chooser/const.ts

```diff
@@ -54,3 +54,8 @@ export const SORTABLE_CONST = {
     drag: 'drag',
   },
 };
+
+export enum SortOrder {
+  descending = 'desc',
+  ascending = 'asc',
+}
```

Comment:
To be more consistent with standards in public d.ts, let's use plain union `type SortOrder = 'desc' | 'asc'`. Or maybe it's better to import from somewhere we already have it

Reply:
Discussed and decided replace enum with const